src/shippingservice/main.go

```diff
@@ -116,6 +133,8 @@ func (s *server) GetQuote(ctx context.Context, in *pb.GetQuoteRequest) (*pb.GetQ
 // ShipOrder mocks that the requested items will be shipped.
 // It supplies a tracking ID for notional lookup of shipment delivery status.
 func (s *server) ShipOrder(ctx context.Context, in *pb.ShipOrderRequest) (*pb.ShipOrderResponse, error) {
+	span := apitrace.SpanFromContext(ctx)
+	span.AddEvent(ctx, "Ship Order")
 	log.Info("[ShipOrder] received request")
```

Comment:
What does this "span" variable do?

Reply:
span = one unit of execution, in this case it represents the ShipOrder call
seeing multiple spans for one trace, e.g. in the "waterfall view" like in GCP Trace console helps us see the "story" of a call across all microservices and see relative latencies/bottlenecks.